docs/shared-content-source/docs/modules/develop/pages/project-structure.adoc

```diff
@@ -88,7 +88,12 @@ Before delving into that, here we list the runtime plugins:
 `CloudflowSparkPlugin`:: SBT plugin for creating Spark Structured Streaming based streamlets.
 `CloudflowFlinkPlugin`:: SBT plugin for creating Flink-based streamlets.
 
-NOTE: Each runtime plugin uses its own default base image. It's `lightbend/akka-base` for Akka, `lightbend/spark` for Spark and `lightbend/flink` for Flink. To customize the base images, set `cloudflowAkkaBaseImage`, `cloudflowSparkBaseImage`, and `cloudflowFlinkBaseImage` in `settings` of the respective runtime plugins in your application `build.sbt`.
+NOTE: Each runtime plugin uses its own default base image. It's `lightbend/akka-base` for Akka, `lightbend/spark` for Spark and `lightbend/flink` for Flink. It could be the case that you're working against your own artifactory registry and therefore prefer to point to a copy or even a customize version of our base images. To customize the base images, set `cloudflowAkkaBaseImage`, `cloudflowSparkBaseImage`, and `cloudflowFlinkBaseImage` in `settings` of the respective runtime plugins in your application `build.sbt`.
```

Comment:
I would change "your own artifactory registry" to "your own Docker registry". Artifactory is a commercial product offered by JFrog.

Reply:
Good point @yuchaoran2011 . What about container registry? wouldn't that be more generic?

Reply:
(I deleted previously this comment but as @yuchaoran2011 answered to it I'm putting it back) wouldn't be better container registry instead of docker one?